src/main.py

```diff
@@ -58,13 +61,17 @@ def execute_block(block_name):
         return 'State must be provided when resuming', HTTPStatus.BAD_REQUEST
 
     # Create new dictionary that will only contain parameters included in block specification
-    cleaned_params = {}
+    cleaned_params = dict()
 
-    requested_params = {}
     if is_resuming and 'requested_params' in body['state']:
         requested_params = body['state']['requested_params']
-    else:
+    elif not is_resuming:
         requested_params = block.get_info()['params']
+    else:
+        requested_params = dict()
```

Comment:
With is_resuming being boolean, can this ever be accessed?

Reply:
Yes - when is_resuming is true, but requested_params does not exist in state (that is, any time we have suspended without requesting any new parameters).

Reply:
Nevermind, can have requested_params missing...